include/libtorrent/netlink.hpp

```diff
@@ -63,7 +63,7 @@ namespace libtorrent
 			std::memset(&m_sockaddr, 0, sizeof(sockaddr_nl));
 			m_sockaddr.nl_family = AF_NETLINK;
 			m_sockaddr.nl_groups = group;
-			m_sockaddr.nl_pid = pid;
+			m_sockaddr.nl_pid = std::uint32_t(pid);
```

Comment:
@arvidn I need some help to decide here. In the android headers, `sockaddr_nl::nl_pid` is hardcoded to `__u32`, in the documentation http://man7.org/linux/man-pages/man7/netlink.7.html it is `pid_t`, but the actual meaning of the field is "Port ID" (not "Process ID").

Then you have `nlmsghdr:: nlmsg_pid` coded to `__u32` in the documentation. Not clear to me what to do